.pre-commit-config.yaml

```diff
@@ -5,25 +5,26 @@ repos:
     - id: pre-commit-config-updater
       always_run: true
 -   repo: https://github.com/asottile/seed-isort-config
-    rev: v1.9.3
+    rev: v2.2.0
     hooks:
     - id: seed-isort-config
 -   repo: https://github.com/pre-commit/mirrors-isort
-    rev: v4.3.21
+    rev: v5.10.1
     hooks:
     - id: isort
       additional_dependencies: [toml]
+      args: ["--profile", "black"]
 -   repo: https://github.com/psf/black
-    rev: 19.10b0
+    rev: 22.3.0
     hooks:
     - id: black
       language_version: python3
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
+-   repo: https://github.com/pycqa/flake8
+    rev: 4.0.1
```

Comment:
I am assuming these are latest versions. Did you get a chance to look at release notes change log because these seem major changes. We could have some features which can be used.

Reply:
Just saying the version for flake8 we were using was released in 2015 and now it is 2022. I am sure there are features we could use

Reply:
Also just saying everyone is dropping support for python 3.6 now. So 3.7 is next, maybe we should upgrade. Just to give context python 3.11 is faster by 20 percent from 3.10 in C-python terms.